src/locales/scripts/wp-locales.json

```diff
@@ -467,7 +467,7 @@
     "facebookLocale": "el_GR",
     "wpLocale": "el",
     "code": "el",
-    "translated": 9
+    "translated": 28
```

Comment:
This looks like the change from #545, should this PR be pointed at that branch?

Reply:
Whoops, I didn't mean to include that file! I'll remove it from the commit.